packages/pool/contracts/GetterUtils.sol

```diff
@@ -277,6 +277,32 @@ abstract contract GetterUtils is StateUtils, IGetterUtils {
         }
     }
 
+    function getUser(address userAddress)
+        external
+        view
+        override
+        returns(
+            uint256 unstaked,
+            uint256 vesting,
+            uint256 unstakeScheduledFor,
+            uint256 unstakeAmount,
+            uint256 mostRecentProposalTimestamp,
+            uint256 mostRecentVoteTimestamp,
+            uint256 mostRecentDelegationTimestamp,
+            uint256 mostRecentUndelegationTimestamp
+            )
+    {
+        User storage user = users[userAddress];
+        unstaked = user.unstaked;
+        vesting = user.vesting;
+        unstakeScheduledFor = user.unstakeScheduledFor;
+        unstakeAmount = user.unstakeAmount;
+        mostRecentProposalTimestamp = user.mostRecentProposalTimestamp;
+        mostRecentVoteTimestamp = user.mostRecentVoteTimestamp;
+        mostRecentDelegationTimestamp = user.mostRecentDelegationTimestamp;
+        mostRecentUndelegationTimestamp = user.mostRecentUndelegationTimestamp;
+    }
```

Comment:
Hmmm, maybe It'll be good to use explicit return here?

Reply:
You mean like a single
```
return;
```
at the end?

Reply:
Yep

Reply:
That would be extremely redundant